### PR TITLE
plugin: hexo-highcharts

### PR DESCRIPTION
**Pull request by lilunze**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
    - Languages:
    - [ ] `en` English
    - [ ] `ko` Korean
    - [ ] `pt-br` Brazilian Portuguese
    - [ ] `ru` Russian
    - [ ] `th` Thai
    - [ ] `zh-cn` simplified Chinese
    - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

File: source/_data/plugins.yml [hunk 1]
```diff
@@ -2723,10 +2723,18 @@
...
+    - highcharts
+    - chart
 - name: hexo-image-cloudflare
   description: Use Cloudflare to cache images in Hexo
   link: https://github.com/njzjz/hexo-image-cloudflare
   tags:
     - image
     - cloudflare
     - cdn
+
```
Comment (SukkaW) [inline source/_data/plugins.yml:2740]: Would you mind removing the extra blank line?

Comment (lilunze) [reply source/_data/plugins.yml:2740]: Can be removed

File: source/_data/plugins.yml [hunk 2]
```diff
@@ -2723,10 +2723,17 @@
...
+    - highcharts
+    - chart
 - name: hexo-image-cloudflare
   description: Use Cloudflare to cache images in Hexo
   link: https://github.com/njzjz/hexo-image-cloudflare
   tags:
     - image
     - cloudflare
-    - cdn
+    - cdn
```
Comment (jiangtj) [inline source/_data/plugins.yml:2739]: Leave a blank line at the end of the file. 😂

File: source/_data/plugins.yml (hunk 1 above)
Comment (jiangtj) [reply source/_data/plugins.yml]: You might add two blank lines, we just hope  you to remove one of these lines 😂

**Review by stevenjoezhang — APPROVED** [review]

**Pull request merged**